src/web_service/cookie.cpp

```diff
@@ -0,0 +1,1052 @@
+            zone_name = "PDT";
+            break;
+        case TT_MST:
+            zone_offset = -7 * 60;
+            zone_name = "MST";
+            break;
+        case TT_MDT:
+            zone_offset = -7 * 60;
+            dst_offset = 1 * 60;
+            zone_name = "MDT";
+            break;
+        case TT_CST:
+            zone_offset = -6 * 60;
+            zone_name = "CST";
+            break;
+        case TT_CDT:
+            zone_offset = -6 * 60;
+            dst_offset = 1 * 60;
+            zone_name = "CDT";
+            break;
+        case TT_EST:
+            zone_offset = -5 * 60;
+            zone_name = "EST";
+            break;
+        case TT_EDT:
+            zone_offset = -5 * 60;
+            dst_offset = 1 * 60;
+            zone_name = "EDT";
+            break;
+        case TT_AST:
+            zone_offset = -4 * 60;
+            zone_name = "AST";
+            break;
+        case TT_NST:
+            zone_offset = -3 * 60 - 30;
+            zone_name = "NST";
+            break;
+        case TT_GMT:
+            zone_offset = 0;
+            zone_name = "GMT";
+            break;
+        case TT_BST:
+            zone_offset = 0;
+            dst_offset = 1 * 60;
+            zone_name = "BST";
+            break;
+        case TT_MET:
+            zone_offset = 1 * 60;
+            zone_name = "MET";
+            break;
+        case TT_EET:
+            zone_offset = 2 * 60;
+            zone_name = "EET";
+            break;
+        case TT_JST:
+            zone_offset = 9 * 60;
+            zone_name = "JST";
+            break;
+        default:
+            break;
+        }
+    }
+
+    /*
+     *  If we didn't find a year, month, or day-of-the-month, we can't
+     *  possibly parse this, and in fact, mktime() will do something random
+     */
+    if (month == TT_UNKNOWN || date == -1 || year == -1 || year > 32767) {
+        return false;
+    }
+
+    memset(result, 0, sizeof(*result));
+    if (sec != -1) {
+        result->tm_sec = sec;
+    }
+    if (min != -1) {
+        result->tm_min = min;
+    }
+    if (hour != -1) {
+        result->tm_hour = hour;
+    }
+    if (date != -1) {
+        result->tm_mday = date;
+    }
+    if (month != TT_UNKNOWN) {
+        result->tm_mon = (((int)month) - ((int)TT_JAN));
+    }
+    if (year != -1) {
+        result->tm_year = year - TM_YEAR_BASE;
+    }
+    if (dotw != TT_UNKNOWN) {
+        result->tm_wday = (((int)dotw) - ((int)TT_SUN));
+    }
+
+    if (zone == TT_UNKNOWN) {
+        // No zone was specified, so pretend the zone was GMT.
+        zone = TT_GMT;
+        zone_offset = 0;
+        zone_name = "GMT";
+    }
+
+    result->tm_isdst = dst_offset * 60;
+    result->tm_gmtoff = zone_offset * 60;
+    result->tm_zone = zone_name;
+
+    return true;
+}
+
+std::pair<std::string, std::string> Cookie::NextPair(const std::string& text, size_t& position,
+                                                     bool isNameValue) {
```

Comment:
```suggestion
                                                     bool is_name_value) {
```

Ditto for other similarly camelCased names below
```